app/_layout.tsx

```diff
@@ -32,6 +35,9 @@ const Layout = () => {
     DMSans_500Medium_Italic,
     DMSans_700Bold,
     DMSans_700Bold_Italic,
+    Inter_400Regular,
+    Inter_500Medium,
+    Inter_600SemiBold,
```

Comment:
Also why are we loading new fonts?

Reply:
These are the fonts used by [Privy UI elements](https://docs.privy.io/guide/expo/ui/).

Reply:
Ohh I see, and theres no way to switch them with the ones we are using?

Reply:
To my knowledge, no.

I'd like to discuss the usage of these UIs tomorrow